--- ccrtp-2.0.3/src/Makefile.am.orig	2012-03-21 15:18:01.000000000 +0100
+++ ccrtp-2.0.3/src/Makefile.am	2012-04-17 20:53:26.361473681 +0200
@@ -22,7 +22,7 @@
 
 if SRTP_OPENSSL
 srtp_src_o =    ccrtp/crypto/openssl/hmac.cpp \
-        ccrtp/crypto/openssl/SrtpSymCrypto.cpp
+        ccrtp/crypto/openssl/SrtpSymCrypto.cpp ccrtp/crypto/twofish.c
 endif
 
 if SRTP_GCRYPT
--- ccrtp-2.0.2/configure.ac.orig	2011-11-13 23:49:08.000000000 +0100
+++ ccrtp-2.0.2/configure.ac	2012-01-03 17:07:33.392407821 +0100
@@ -95,7 +95,7 @@
                   ],
                   [AC_CHECK_LIB([crypto],
                       [EVP_CipherInit_ex],
-                      [AC_CHECK_HEADERS([openssl/bn.h openssl/aes.h openssl/sha.h],
+                      [AC_CHECK_HEADERS([openssl/bn.h openssl/aes.h openssl/sha.h pthread.h],
                                         [], AC_MSG_ERROR([OpenSSL headers not found.]))
                        LIBS="-lcrypto $LIBS"
                        openssl="true"
